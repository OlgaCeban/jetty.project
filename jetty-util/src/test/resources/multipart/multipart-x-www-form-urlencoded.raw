--qjIkwQOhaYfC2VEcL5j-9sjEK1FIsYMd5
Content-Disposition: form-data; name="company"
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
Content-Transfer-Encoding: 8bit

bob+%26+frank%27s+shoe+repair
--qjIkwQOhaYfC2VEcL5j-9sjEK1FIsYMd5--
