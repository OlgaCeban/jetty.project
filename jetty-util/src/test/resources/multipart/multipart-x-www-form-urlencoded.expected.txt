Content-Type|multipart/form-data; boundary="qjIkwQOhaYfC2VEcL5j-9sjEK1FIsYMd5"
Parts-Count|1
Part-ContainsContents|company|bob & frank's shoe repair


